substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -1944,6 +1944,8 @@ pub mod pallet {
 		NothingToAdjust,
 		/// No slash pending that can be applied to the member.
 		NothingToSlash,
+		/// The slash amount is too low to be applied.
+		SlashTooLow,
```

Comment:
This is a breaking change, but we have already backported this to [2407](https://github.com/paritytech/polkadot-sdk/pull/6591) and [2409](https://github.com/paritytech/polkadot-sdk/pull/6592), so probably better keep it that way.